<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcceptRequest</class>
 <widget class="QWidget" name="AcceptRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Passenger Name :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="Passenger">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="FindBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Info">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>160</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>c4.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>Passenger</zorder>
  <zorder>FindBtn</zorder>
  <zorder>Info</zorder>
  <zorder>AcceptBtn</zorder>
  <zorder>CancelBtn</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
